SWLOR.Game.Server/Feature/AbilityDefinition/Devices/WristRocketAbilityDefinition.cs

```diff
@@ -69,13 +69,14 @@ private void WristRocket1()
                 .Name("Wrist Rocket I")
                 .Level(1)
                 .HasRecastDelay(RecastGroup.WristRocket, 24f)
-                .HasActivationDelay(2f)
+                .HasActivationDelay(0f)
                 .RequirementStamina(1)
                 .UsesAnimation(Animation.CastOutAnimation)
                 .IsCastedAbility()
                 .UnaffectedByHeavyArmor()
                 .BreaksStealth()
                 .HasMaxRange(15f)
+                .IsHostileAbility()
```

Comment:
This helps with not accidentally targeting friendlies, but the AOE will still affect friendlies unless explicitly ignored. Is this set up like you expect?